java/src/jmri/jmrit/display/layoutEditor/LayoutTrack.java

```diff
@@ -106,5 +101,24 @@ public void setHidden(boolean hide) {
         hidden = hide;
     }
 
+    /**
+     * non-accessor methods
```

Comment:
The comment syntax needs to change on this, as Javadoc will interpret this as a comment on the next method.